django_filters/filterset.py

```diff
@@ -336,7 +336,11 @@ def filter_for_field(cls, f, name, lookup_expr='exact'):
             "https://django-filter.readthedocs.io/en/develop/ref/filterset.html#customise-filter-generation-with-filter-overrides"
         ) % (cls.__name__, name, lookup_expr, f.__class__.__name__)
 
-        return filter_class(**default)
+        filt = filter_class(**default)
+        if (not settings.DISABLE_HELP_TEXT and
```

Comment:
The `DISABLE_HELP_TEXT` check shouldn't be necessary, since it's performed in [`Filter.field`](https://github.com/carltongibson/django-filter/blob/1.0.2/django_filters/filters.py#L128-L129).